core/src/main/scala/org/apache/spark/executor/ExecutorSource.scala

```diff
@@ -95,7 +100,8 @@ class ExecutorSource(threadPool: ThreadPoolExecutor, executorId: String) extends
   })
 
   // Expose executor task metrics using the Dropwizard metrics system.
-  // The list is taken from TaskMetrics.scala
+  // The list of available Task metrics is available in TaskMetrics.scala
```

Comment:
Nit: This sentence is strange, because of the two available words: "available Task metrics is available".